recipes/dbus/1.x.x/test_package/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(DBus1 REQUIRED)
+
+add_executable(example example.cpp)
```

```suggestion
add_executable(example example.c)
```
Can it be consumed from C projects ?